src/components/Filters/FilterRow/FilterRow.types.ts

```diff
@@ -0,0 +1,11 @@
+import { DataPoint, Filter } from '../Filters.types';
+
+export interface SplitFieldProps {
+  width?: number;
+}
+
+export interface FilterRowProps extends Filter {
+  id: string;
+  dataPoints: DataPoint[];
+  isFirstRow?: boolean;
```

Comment:
this prop isn't defined in propTypes and is not even used in the component

Reply:
good catch! i missed that, will remove it